pr/webhook/server.go

```diff
@@ -51,35 +53,50 @@ func (receiver *Receiver) Start() {
 			return
 		}
 
+		receiver.wg.Add(1)
```

Comment:
This seems late in the request handling. Ideally, accepting a new request and increasing the `receiver.wg` semaphore would happen "at the same time". In this case, I guess that's not easily possible because the ServeMux actually accepts the connection, but then we should probably at least move the `receiver.wg.Add(1)` to the beginning of this function.